db/migrate/20240510033206_sanitize_enterprise_long_description.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class SanitizeEnterpriseLongDescription < ActiveRecord::Migration[7.0]
+  class Enterprise < ApplicationRecord
+  end
+
+  def up
+    Enterprise.where.not(long_description: [nil, ""]).find_each do |enterprise|
+      enterprise.update!(long_description: sanitize(long_description))
+    end
+  end
```

Comment:
This is un-reversable of course, so I think it's a bit risky (what if we messed up formatting for lots of enterprises).
Maybe it would be fine, and I don't want to make more effort unnecessarily, but would consider first sanitising the view and wait until it's been released for a couple of weeks before applying the migration.

Reply:
Good point. We should at least test it first. The `br` tag is a good example for something I forgot and where I'm not sure it's covered.